Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15863,6 +15863,10 @@ INSTANTIATE_SEND_TO_PROCESS_CONTAINING_FRAME(WebPage::LoadURLInFrame);
 INSTANTIATE_SEND_TO_PROCESS_CONTAINING_FRAME(WebPage::LoadDataInFrame);
 INSTANTIATE_SEND_TO_PROCESS_CONTAINING_FRAME(WebProcess::BindAccessibilityFrameWithData);
 INSTANTIATE_SEND_TO_PROCESS_CONTAINING_FRAME(WebPage::UpdateFrameScrollingMode);
+#if PLATFORM(MAC)
```

Comment:
Maybe?
```suggestion
#if ENABLE(PDF_HUD)
```

Reply:
Actually, ignore my suggestion, it looks like this has other uses besides the HUD.